tests/console/check_os_release.pm

```diff
@@ -14,13 +14,16 @@ use strict;
 use warnings;
 use testapi;
 use version_utils qw(is_sle is_leap is_tumbleweed is_sles4sap is_rt is_hpc);
-use main_common 'is_desktop';
+use main_common qw(is_desktop is_migration_tests);
 
 sub run {
     my ($self) = @_;
     $self->select_serial_terminal;
 
     my %checker = ();
+    if (is_migration_tests) {
+        $checker{TARGET_VERSION} = get_var("UPGRADE_TARGET_VERSION", get_required_var("VERSION"));
```

Comment:
As my understanding, this should be:
$checker{VERSION}  = get_var("TARGET_VERSION", get_required_var("VERSION"));

And remove L27 WDYT?

Reply:
OK. thanks